EngineLayer/Psm.cs

```diff
@@ -401,14 +401,14 @@ internal void Add(Psm psmParent)
         private Tuple<string, double?> Resolve(IEnumerable<double> enumerable)
         {
             double doubleTol = 1e-6;
-            var list = enumerable.ToList();
+            var list = enumerable.Distinct().ToList();
```

Comment:
Same protein repeated several times per peptide is an extremely rare occurrence. This means that somehow there is a repeating sequence, and that is interesting in itself. I wouldn't want to hide it!